ui/src/app/features/collection-manager-subject-left/collection-manager-subject-left.container.component.ts

```diff
@@ -55,6 +61,12 @@ export class CollectionManagerSubjectLeftContainerComponent implements OnInit {
     this.subjectsList$ = this.collectionLeftFacade.subjectsList$;
     this.isPending$ = this.collectionLeftFacade.isPending$;
     this.apiKey$ = this.collectionLeftFacade.apiKey$;
+    this.collectionItemsSubs = combineLatest([this.currentSubject$, this.collectionRightFacade.collectionItems$])
+      .pipe(
+        map(([, collection]) => collection.filter(item => item.status === CircleLoadingProgressEnum.InProgress)),
```

Comment:
is there a need for `combineLatest` since we are not using the result of this.currentSubject$?

Reply:
fixed